src/page/User/Logout.tsx

```diff
@@ -1,4 +1,22 @@
+import { useEffect } from 'react';
+import useApi from '../../hook/useApi';
+import { useNavigate } from 'react-router-dom';
+
 function Logout() {
+  const { userApi } = useApi();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    userApi
+      .post<any>('/logout')
+      .then(() => {
+        navigate('/');
+      })
+      .catch(() => {
+        alert('Logout Failed!');
+        navigate('/');
+      });
```

Comment:
```suggestion
      .catch(() => {
        alert('Logout Failed!');
      })
      .finally(() => {
      	navigate('/');
      });
```
finally를 쓰면 항상 then이나 catch 이후에 실행됩니다.

Reply:
알겠습니다 좋은 정보 감사합니다.